### PR TITLE
Updated example, created jupyer notebook

### PR DESCRIPTION
**Pull request by abachma2**
Updated the current example to reflect changes in cymetric metrics since the problem was created (addressing issue #172). Created jupyter notebook with the updated example (addressing issues #103 and #134). Added creation of plot of total reactor prototypes as a function of time step.

### CODE REVIEW

**Review by bam241 — APPROVED** [review]
LGTM (Looks Good To Me) !
Thank you very much @abachma2 .

I'll be merging tonight, if no-one request modifications....

**Comment by bam241** [comment]
Just for the future, it is usually a best practice to create a dedicate branch to work on modification, new features etc.... To keep the "mess" separated from you master branch, that should remain an clean copy of the cyclus/cymetric master branch. 
(let me know if any of this does not make sense :) )

**Comment by abachma2** [comment]
My bad. I need to get used to doing that.
Do @katyhuff or @gwenchee like to take a look before it gets merged?

File: examples/agent_entryexit.ipynb [hunk 1]
```diff
@@ -0,0 +1,2608 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cymetric Example for Build and Decommissioning of Prototypes\n",
+    "\n",
+    "This example walked through how to plot the number of reactor prototypes built during the simulation. \n",
```
Comment (katyhuff) [inline examples/agent_entryexit.ipynb:9]:
```suggestion
    "This example walks through how to plot the number of reactor prototypes built during the simulation. \n",
```

**Review by katyhuff — CHANGES_REQUESTED** [review]
minor wording change in notebook. 
Two files in the examples folder were deleted as part of this PR. Was that intentional?

**Comment by abachma2** [comment]
Are you referring to the `ex.png` and `recycle.sqlite` as the files that have been removed?

**Comment by katyhuff** [comment]
Ah yes, NVM. They were just changed. (my bad... silly phone interface)

**Review by katyhuff — APPROVED** [review]

**Comment by katyhuff** [comment]
Thanks @abachma2 @bam241 !! Merging.

**Pull request merged**